### PR TITLE
move cleanupContainerImages to after ensureDocker

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/cse_main.sh [hunk 1]
```diff
@@ -159,6 +158,11 @@ ensureContainerd {{/* containerd should not be configured until cni has been con
 ensureDocker
 {{- end}}
 
+if [ -f $VHD_LOGS_FILEPATH ]; then
+    echo "removing unused pre-pulled images"
+    cleanUpContainerImages
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/cse_main.sh:163]: remove L79

Comment (xuto2) [reply parts/linux/cloud-init/artifacts/cse_main.sh:163]: should we be worried about prolonged cse execution due to the delay or disk space issue (in extreme case when vhd is almost full and we cannot install anything before cleaning up the images)? I have two possible options
1) move ensureDocker/containerd up to as the first step, but I notice containerd has dependencies on CNI, not sure if there are other dependencies that should be moved along too
2) keep the current cleanup image and redo it after ensureDocker/containerd?

**Comment by xuto2** [comment]
Do we still need this change?

**Comment by alexeldeib** [comment]
all vhds come with an appropriate container runtime preinstalled, it just might not be the right version. I suppose that could cause errors during image cleanup during reinstall of the correct version. In practice it works fine from most logs i've seen, so going to close.

**Pull request closed without merging**